lab-justin/test/storage-unit-test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Storage = require('../lib/storage');
+const testStoreage = new Storage(`${__dirname}/data`);
+
+describe('testing storage module', function(){
+  describe('testing setItem', function(){
+    it('should return the item', function(done){
+      testStoreage.setItem('note', {id: 123456, content: 'test test test'})
+      .then(function(item){
+        expect(item.id).to.equal(123456);
+        expect(item.content).to.equal('test test test');
+        done();
+      }).catch(function(err){
+        console.error(err);
+        expect(err).to.equal(undefined);
+        done();
+      });
+    });
+  });
+});
+
+describe('testing storage module', function(){
+  describe('testing fetchItem', function(){
+    it('should retrieve the item', function(done){
+      testStoreage.setItem('note', {id: 123456, content: 'test test test'})
```

Comment:
This is a fetchItem test, so it shouldn't be testing setItem. As is, this test is redundant with the one above it.